src/dynarec/arm64/dynarec_arm64_avx_0f38.c

```diff
@@ -150,13 +227,14 @@ uintptr_t dynarec64_AVX_0F38(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, i
             GETED(0);
             GETVD;
             UXTBw(x1, vd);
-            CMPSw_U12(x1, rex.w?64:32);
-            CSETxw(x2, cPL);
+            CMPSw_U12(x1, rex.w?63:31);
+            CSETxw(x2, cGT);
             IFX(F_CF) {
                 BFIw(xFlags, x2, F_CF, 1);
             }
-            MVNxw_REG(x2, x2); //prepare mask
-            B_MARK(cPL);
+            MOV64xw(x2, 0); //prepare mask
+            B_MARK(cGT);
+            MOV64xw(x2, -1);
```

Comment:
why this?

Reply:
The code that was there is stricly equivalent to the one you put, but shorter.

x2 is `-1` if x1>=64, while it's `0` else, because of the CSET before...
after the MVN it become `0` if x1>=64, and `-1` else
then it jump to MARK if x1>=64

Reply:
Ah, the mystake is that it should be `CSETM` instead of `CSET` for this to work. (CSETM set all bits to 1, while CSET set the value to 1)